apps/studio/src/components/PageEditor/Editor.tsx

```diff
@@ -61,32 +62,36 @@ const renderers: JsonFormsRendererRegistryEntry[] = [
 
 const Editor = ({ jsonSchema, editorValue, onChange }: any) => {
   return (
-    <Box>
-      <ThemeProvider theme={jsonFormsTheme}>
-        <CssBaseline />
-        <Box
-          padding="0.5rem"
-          overflowY={'auto'}
-          maxH="full"
-          css={{
-            '& div': {
-              maxWidth: '100% !important', // Setting max-w-full with high specificity
-            },
-            '& div.MuiTabs-flexContainer': {
-              overflowX: 'auto',
-            },
-          }}
-        >
-          <JsonForms
-            schema={jsonSchema}
-            data={editorValue}
-            renderers={renderers}
-            cells={materialCells}
-            onChange={({ data }) => onChange(data)}
-          />
-        </Box>
-      </ThemeProvider>
-    </Box>
+    <ComponentSelector
+      onClose={() => console.log('close')}
+      onProceed={(componentType) => console.log(componentType)}
+    />
+    // <Box>
+    //   <ThemeProvider theme={jsonFormsTheme}>
+    //     <CssBaseline />
+    //     <Box
+    //       padding="0.5rem"
+    //       overflowY={'auto'}
+    //       maxH="full"
+    //       css={{
+    //         '& div': {
+    //           maxWidth: '100% !important', // Setting max-w-full with high specificity
+    //         },
+    //         '& div.MuiTabs-flexContainer': {
+    //           overflowX: 'auto',
+    //         },
+    //       }}
+    //     >
+    //       <JsonForms
+    //         schema={jsonSchema}
+    //         data={editorValue}
+    //         renderers={renderers}
+    //         cells={materialCells}
+    //         onChange={({ data }) => onChange(data)}
+    //       />
+    //     </Box>
+    //   </ThemeProvider>
+    // </Box>
```

Comment:
nit: can remove

Reply:
Removed in https://github.com/opengovsg/isomer/pull/187/commits/71703b3913474b61c929c028c71dea8f069681c4